src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <PropertyGroup>
```

Comment:
WCF test projects have project references to S.P.SM.csproj, the default configuration for these projects was just “Debug” and in their reference to S.P.SM.csproj they did not specify “OSGroup”, therefore; an AnyOS build of S.P.SM.csproj was being triggered.

The race condition comes into play when the individual test projects are being created and the test assembly together with any dependent assemblies are being copied/bin-placed specifically in the wcf\bin\AnyOS.AnyCPU.Debug folder. It looks like for those test folders created while the AnyOS version of S.P.SM has not yet completed building it will find the Windows version already built and copy that to the test folder then once the AnyOS version of S.P.SM has completed building the remaining test folders get that version of S.P.SM.

In Ron’s case a test project with tests of a feature that only exists in the Windows build had the AnyOS version of S.P.SM, in my case (probably because my machine is slower and the AnyOS version of S.P.SM had not finished building) the Windows version of S.P.SM was in that particular test folder.
By the end of the process however both of us had the AnyOS version of S.P.SM in …\wcf\bin\ AnyOS.AnyCPU.Debug\ System.Private.ServiceModel.

I don’t have a solution for the race condition but we can avoid this entirely by adding the default “Windows_Debug” back to the S.P.SM.proj file for which I am creating a PR. In addition, I am updating all of our test projects and other supporting projects with this same default, this way we won’t have an “AnyOS” bin directory at all and it will save a little time when we build locally.

Regarding Eric’s comments in the other PR and whether or not there is another race condition related to having that default setting in S.P.SM we can discuss in the new PR.